doc/advanced-topics.md

```diff
@@ -1,5 +1,25 @@
 ## Advanced Topics
 
+* [Secured access to the Context Broker](#secured-access-to-the-context-broker)
+* [GeoJSON support this only applies for NGSI LD, not for NGSIv1 and NGSIv2](#geojson-support-this-only-applies-for-ngsi-ld-not-for-ngsiv1-and-ngsiv2)
```

Comment:
It seems linter doesn't like this line:

![imagen](https://user-images.githubusercontent.com/1534240/121416612-47e6f700-c969-11eb-8429-498ccf8b0698.png)

If changed, I understand the actual title in line 69 should be also be changed.

Reply:
Or maybe better "NGSI-v2"...

Not sure which is better ("NGSI v2" or "NGSI-v2") from a English language point of view. Check other uses in existing documentation.

Reply:
In other parts in this repo is used both NGSI-v2 and NGSI v2, but seems more used the first one. On Orion repo is used NGSIv2.